documentation/src/main/asciidoc/mapper-orm-mapping.asciidoc

```diff
@@ -174,6 +174,11 @@ Whether the field can be <<mapper-orm-query-projection, projected on>>,
 i.e. whether the field value is stored in the index to allow later retrieval when querying.
 +
 Value: `Projectable.YES`, `Projectable.NO`, `Projectable.DEFAULT`.
+`searchable`::
+Whenever the field can be searched on.
+i.e. whether the field is indexed. So the document will be searchable using this field, using it in the query predicates.
```

Comment:
1. A sentence may start with a conjunction, such as "so", but frankly it feels weird in the (relatively formal) context of documentation.
2. You're basically saying "Searchable: whether the field can be searched on, i.e. whether the field is indexed, so the document will be searchable". A bit confused, don't you think?

Try something like this:

```
i.e. whether the field is indexed in order to allow applying predicates later when querying.
```

Reply:
it sounds better